atom/browser/api/atom_api_app.cc

```diff
@@ -505,6 +507,19 @@ App::App(v8::Isolate* isolate) {
   static_cast<AtomBrowserClient*>(AtomBrowserClient::Get())->set_delegate(this);
   Browser::Get()->AddObserver(this);
   content::GpuDataManager::GetInstance()->AddObserver(this);
+  content::BrowserChildProcessObserver::Add(this);
+  int pid = 0;
```

Comment:
I think you can use the chrome helper for this:

```cpp
ProcessId pid = base::GetCurrentProcId();
```

Then you won't need the two paths.